doc/classifying-signatures.md

```diff
@@ -0,0 +1,107 @@
+# Classifying signatures: `search`, `gather`, and `lca` methods.
+
+sourmash provides several different techniques for doing
+classification and breakdown of signatures.
+
+## Searching for similar samples with `search`.
+
+The `sourmash search` command is most useful when you are looking for
+high similarity matches to other signatures; this is the most basic use
+case for MinHash searching.  The command takes a query signature and one
+or more search signatures, and finds all the matches it can above a particular
+threshold.
+
+By default `search` will find matches with high [*Jaccard
+similarity*](https://en.wikipedia.org/wiki/Jaccard_index), which will
+consider all of the k-mers in the union of the two samples.
+Practically, this means that you will only find matches if there is
+both high overlap between the samples *and* relatively few k-mers that
+are disjoint between the samples.  This is effective for finding genomes
+or transcriptomes that are similar but rarely works well for samples
+of vastly different sizes.
+
+One useful modification to `search` is to calculate containment with
+`--containment` instead of the (default) similarity; this will find
+matches where the query is contained within the subject, but the
+subject may have many other k-mers in it. For example, if you are using
+a plasmid as a query, you would use `--containment` to find genomes
+that contained that plasmid.
+
+See [the main sourmash
+tutorial](http://sourmash.readthedocs.io/en/latest/tutorials.html#make-and-search-a-database-quickly)
+for information on using `search` with and without `--containment`.
+
+## Breaking down metagenomic samples with `gather` and `lca`
+
+Neither search option (similarity or containment) is effective when
+comparing or searching with metagenomes, which typically have a
+mixture of many different genomes.  While you might use containment to
+see if a query genome is present in one or more metagenomes, a common
+question to ask is the reverse: **what genomes are in my metagenome?**
+
+We have implemented two algorithms in sourmash to do this.
+
+One, approaches based on lowest common ancestor ("LCA"), uses
```

Comment:
The language here is a little confusing, but not that critical to change

Reply:
Fixed! Thanks for noticing!